lib/vmdb/appliance.rb

```diff
@@ -146,24 +146,40 @@ def self.get_network
       retVal
     end
 
-    def self.init_diagnostics
-      log_dir = "/var/www/miq/vmdb/log"
-      gem_log = File.join(log_dir, "gem_list.txt")
-      rpm_log = File.join(log_dir, "package_list_rpm.txt")
+    def self.installed_rpms
+      File.open(File.join(log_dir, "package_list_rpm.txt"), "a") do |file|
+        puts file, "start: date time is: #{Time.now.utc}"
+        LinuxAdmin.Rpm.list_installed.sort do |name, version|
```

Comment:
`list_installed` returns a hash.  I don't think you can sort a hash directly like this.  You would have to do something like:

```ruby
Hash[LinuxAdmin.Rpm.list_installed.sort].each do |name, version|
  #stuff
end
```